File: solutions/extra-task-1/src/main/java/io/miragon/training/application/port/outbound/MembershipProcess.java
package io.miragon.training.application.port.outbound;

import io.miragon.training.domain.Membership;
import io.miragon.training.domain.MembershipId;

public interface MembershipProcess {

    void startProcess(Membership membership);

    void rejectMembership(MembershipId membershipId);
}

File: solutions/exercise-5/src/main/kotlin/io/miragon/training/application/port/outbound/MembershipProcess.kt
package io.miragon.training.application.port.outbound

import io.miragon.training.domain.Membership
import io.miragon.training.domain.MembershipId

interface MembershipProcess {

    fun startProcess(membership: Membership)

    fun rejectMembership(membershipId: MembershipId)
}
